### PR TITLE
Add more container metrics from mesos

### PR DESCRIPTION
**Pull request by bergerx**
This patch will introduce more container metrics available from Mesos to be
exposed. DCOS_OSS-1607 has more deatils about metrics exposed and link to the
related definitions in Mesos.

https://jira.mesosphere.com/browse/DCOS_OSS-1607

### CODE REVIEW

**Comment by bergerx** [comment]
This one follows contribution guidelines and passes all the existing unit tests. I don't see a specific reason to add extra tests for this.

File: collectors/mesos/agent/container.go [hunk 1]
```diff
@@ -48,15 +48,32 @@ type agentContainer struct {
...
-	MemTotalBytes uint64 `json:"mem_total_bytes,omitempty"`
-	MemLimitBytes uint64 `json:"mem_limit_bytes,omitempty"`
+	MemTotalBytes              uint64 `json:"mem_total_bytes,omitempty"`
+	MemTotalMemswBytes         uint64 `json:"mem_total_memsw_bytes,omitempty"`
+	MemLimitBytes              uint64 `json:"mem_limit_bytes,omitempty"`
+	MemSoftLimitBytes          uint64 `json:"mem_soft_limit_bytes,omitempty"`
+	MemCacheBytes              uint64 `json:"mem_cache_bytes,omitempty"`
+	MemRssBytes                uint64 `json:"mem_rss_bytes,omitempty"`
+	MemMappedFileBytes         uint64 `json:"mem_mapped_file_bytes,omitempty"`
+	MemSwapBytes               uint64 `json:"mem_swap_bytes,omitempty"`
```
Comment (bergerx) [inline collectors/mesos/agent/container.go]: I know swap is not enabled by default in DC/OS, but this doesn't imply that no one will enable swap to address some specific requirements. So I left these ones as well.

**Comment by bergerx** [comment]
@philipnrmn can you have a look at this one?

**Comment by philipnrmn** [comment]
OK to test.

**Comment by philipnrmn** [comment]
Before:
```
$ dcos task metrics details statsd-emitter-old.0aaa0887-b05a-11e7-9a90-f6ce329d9007                                                   VALUE
cpus.system.time                                       0.03
cpus.user.time                                         0.12
cpus.throttled.time                                    10.41
net.tx.errors                                          0
net.rx.bytes                                           0.00GiB
mem.limit                                              0.16GiB
net.tx.dropped                                         0
net.rx.errors                                          0
net.tx.packets                                         0
disk.limit                                             0.00GiB
disk.used                                              0.00GiB
net.rx.packets                                         0
net.tx.bytes                                           0.00GiB
net.rx.dropped                                         0
mem.total                                              0.01GiB
cpus.limit                                             0.10
statsd_tester.time.uptime                              1554427
dcos.metrics.module.container_received_bytes_per_sec   64
dcos.metrics.module.container_throttled_bytes_per_sec  0
```

After:

```
$ dcos task metrics details statsd-emitter.09acbcc6-b05a-11e7-9a90-f6ce329d9007
NAME                                                   VALUE
mem.swap                                               0.00GiB
cpus.nr.periods                                        3811
mem.mapped.file                                        0.00GiB
net.rx.packets                                         0
mem.medium.pressure                                    0
mem.total                                              0.01GiB
disk.limit                                             0.00GiB
net.rx.bytes                                           0.00GiB
mem.total.memsw                                        0.00GiB
net.tx.errors                                          0
mem.limit                                              0.16GiB
cpus.limit                                             0.10
processes                                              0
cpus.system.time                                       0.04
net.tx.bytes                                           0.00GiB
cpus.user.time                                         0.12
net.tx.dropped                                         0
disk.used                                              0.00GiB
net.rx.dropped                                         0
threads                                                0
mem.cache                                              0.00GiB
mem.critical.pressure                                  0
mem.soft.limit                                         0.00GiB
mem.low.pressure                                       0
mem.rss                                                0.01GiB
net.tx.packets                                         0
cpus.nr.throttled                                      195
mem.unevictable                                        0.00GiB
cpus.throttled.time                                    13.03
net.rx.errors                                          0
dcos.metrics.module.container_throttled_bytes_per_sec  0
dcos.metrics.module.container_received_bytes_per_sec   64
statsd_tester.time.uptime                              1565795
```

Really nice! Thanks @bergerx!

**Pull request merged**